Explorer/Assets/DCL/FeatureFlags/FeatureFlagsProviderExtensions.cs

```diff
@@ -14,32 +14,40 @@ public static class FeatureFlagsProviderExtensions
             CancellationToken ct)
         {
             FeatureFlagOptions options = FeatureFlagOptions.ORG;
-            URLDomain? programArgsUrl = GetUrlFromProgramArgs();
+            GetOptionsFromProgramArgs(out URLDomain? programArgsUrl, out string? programArgsHostName);
 
             if (programArgsUrl != null)
                 options.URL = programArgsUrl.Value;
 
+            if (programArgsHostName != null)
+                options.Hostname = programArgsHostName;
+
             options.UserId = userAddress;
 
             return await featureFlagsProvider.GetAsync(options, ct);
 
             // #!/bin/bash
-            // ./Decentraland.app --feature-flags-url https://feature-flags.decentraland.zone
-            URLDomain? GetUrlFromProgramArgs()
+            // ./Decentraland.app --feature-flags-url https://feature-flags.decentraland.zone --feature-flags-hostname localhost
+            void GetOptionsFromProgramArgs(out URLDomain? url, out string? hostname)
             {
+                url = null;
+                hostname = null;
+
                 string[] programArgs = Environment.GetCommandLineArgs();
-                URLDomain? result = null;
 
                 for (var i = 0; i < programArgs.Length - 1; i++)
                 {
-                    string arg = programArgs[i];
-
-                    if (arg == "--feature-flags-url")
-                        result = URLDomain.FromString(programArgs[i + 1]);
+                    switch (programArgs[i])
+                    {
+                        case "--feature-flags-url":
```

Comment:
It might be minor, but maybe we could consider making the literals constants?

Reply:
Done!